atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransactionManager.java

```diff
@@ -85,7 +85,7 @@ public void registerClosingCallback(Runnable closingCallback) {
      * use the delegate instead.
      */
     protected SerializableTransactionManager() {
-        this(null, null, null, null, null, null, null, null, null, 1);
+        this(null, null, null, null, null, null, null, null, null, 1, AtlasDbConstants.DEFAULT_TIMESTAMP_CACHE_SIZE);
```

Comment:
all of the transaction manager code is the worst, in case this line doesn't make things clear :-/

Reply:
There's no reason to maintain a large cache for the dummy transaction manager. We can just do `this(null, null, null, null, null, null, null, null, null, 1, 1);`?